### ISSUE TITLE
Microsoft Teams webhook fails after certificate creation (reason: empty payload)

### ISSUE THREAD

**Issue by diakonos** [shibayan/keyvault-acmebot#350]
**Describe the bug**
Microsoft Teams webhook fails after certificate creation with error:
> Failed invoke webhook. Status Code = BadRequest, Reason = Invalid webhook request - Empty Payload

**To Reproduce**
1. Follow setup instructions and configure a Microsoft Teams webhook.
2. Create a new certificate.
3. Search the application insights for "webhook".
4. See webhook error that occurred after certificate creation.

**Environment (please complete the following information):**
 - Certificate Type: Zone Apex and www
 - Certificate Deploy Target: Front Door

**Additional context**
I noticed that Teams recently changed the format of their webhook URL. It now has the format: `https://xxx.webhook.office.com/webhookb2/xxx/IncomingWebhook/xxx/xxx`. Previously it was: `https://outlook.office.com/webhook/xxx/IncomingWebhook/xxx/xxx`. I checked the source code for this project which seems to be looking for .office.com, so I don't believe the URL format change would negatively impact webhook execution.

**Comment by shibayan** [comment]
@diakonos 
Thanks for the report. Did you complete the issuance of the certificate correctly?

**Comment by diakonos** [comment]
Yep, the certificate was issued successfully. Everything else besides the webhook seems to be behaving correctly so far.

**Comment by shibayan** [comment]
@diakonos Thanks. I was able to reproduce this in my environment and I will investigate and fix it.

**Comment by GeorgDangl** [comment]
Just FYI, I've got the same with MS teams, from Application Insights:

```
<html>
<body>
<!--StartFragment-->

Failed invoke webhook. Status Code = BadRequest, Reason = Invalid webhook request - Empty Payload |  
-- | --


<br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html>
```

**Comment by GeorgDangl** [comment]
And as far as I remember, it never worked for me. My main subscription is pushing to Slack, but this one was recently set up a few weeks ago and I've set up a teams webhook.

**Comment by shibayan** [comment]
This issue has been fixed in `v3.6.6`. Thank you!
